portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,92 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>more about B</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="title">
+      <h1>a page about me</h1>
+      <p>this is my portfolio, a little more about B</p>
     </div>
+    <br>
+
+    <div id="about-sidebar">
+        <div class="box">
```

Comment:
nit - this css class `box` doesn't exist yet. let's wait to add it until it's defined.